src/content/app/tools/blast/components/blast-species-selector/BlastSpeciesSelector.scss

```diff
@@ -13,6 +13,10 @@
   background-color: $white;
 }
 
+.smallScreenHeader {
+  top: 49px;
+}
```

Comment:
Same question as above

Reply:
@azangru  This and all the above top margin are related with the position sticky when you scroll the page. The button should be fixed at the top when you scroll the page.

Reply:
Thanks, but I am still confused. `.mainContainerSmall .header` and `.smallScreenHeader` seem to target the same DOM element; why the duplication of the CSS rule?

![image](https://user-images.githubusercontent.com/6834224/168277339-a94a098c-0c8f-4005-bc16-6d5c3e2ee281.png)

Reply:
@azangru you are right it is an extra rule that we dont need, i have removed it.